public/external-text/default.yaml

```diff
@@ -730,7 +730,9 @@ help:
         ${$$.components.gcmDefn}
 
         In PCEX, models are indentified by short codes.
```

Comment:
Maybe somewhere like here would be a good place to explain the difference between CMIP5 and CMIP6? I could also go in the FAQs, though.